src/pollution_model/run_medusa_2.py

```diff
@@ -137,44 +137,56 @@ def compute_tss_roof_road(surface_area: float,
     Returns
     -------
     float
-       Returns the TSS value from the given parameters
+       Returns the TSS value from the given parameters (grams)
 
     Raises
     ----------
     ValueError
         If the surface type is not a roof or road (concrete, copper, galvanised roof, asphalt road, or car park).
     """
-    # Define the constants (Cf is the capacity factor).
-    capacity_factor = 0.75 if surface_type in ROOF_SURFACE_TYPES else 0.25
-    # Values a1 to a3 are empirically derived coefficient values.
-
     # Define error message (if needed)
     invalid_surface_error = (f"Given surface is not valid for computing TSS."
                              f" Needed a roof or road, but got {SurfaceType(surface_type).name}.")
 
     match surface_type:
         case SurfaceType.COLOUR_STEEL:
-            a1, a2, a3 = 0.4, 0.5, 0.008
+            # Galvanised Painted
+            a1, a2, k = 0.4, 0.5, 0.00933
         case SurfaceType.GALVANISED:
-            a1, a2, a3 = 0.4, 0.5, 0.008
+            # Galvanised
+            a1, a2, k = 0.4, 0.5, 0.00933
         case SurfaceType.METAL_OTHER:
+            # Copper Old
             # Using coefficients of Galvanised rather than Copper
-            a1, a2, a3 = 0.4, 0.5, 0.008
+            a1, a2, k = 0.4, 0.5, 0.00933
         case SurfaceType.METAL_TILE:
-            a1, a2, a3 = 0.4, 0.5, 0.008
+            # Decramastic
+            a1, a2, k = 0.4, 0.5, 0.00933
         case SurfaceType.NON_METAL:
-            a1, a2, a3 = 0.6, 0.25, 0.008
+            # Concrete, Concrete Painted, and Glass Roof
+            a1, a2, k = 0.6, 0.25, 0.00933
         case SurfaceType.ZINCALUME:
-            a1, a2, a3 = 0.4, 0.5, 0.008
+            # Zincalume
+            a1, a2, k = 0.4, 0.5, 0.00933
         case SurfaceType.ASPHALT_ROAD | SurfaceType.CAR_PARK:
-            a1, a2, a3 = 190, 0.16, 0.0125
+            a1, a2, a7, k = 190, 0.16, 0.0125, 0.0008
         case _:
             raise ValueError(invalid_surface_error)
+
+    # Call out necessary variables
     antecedent_dry_days, average_rain_intensity, event_duration, _ph = rainfall_event
-    first_term = surface_area * a1 * antecedent_dry_days ** a2 * capacity_factor
-    second_term = 1 - math.exp(a3 * average_rain_intensity * event_duration)
+    # THE K WAS KEPT NEGATIVE IN THE MATH.EXP() FOR HAVING POSITIVE RESULTS
+    both_factor = 1 - math.exp(-k * average_rain_intensity * event_duration)
```

Comment:
I am not sure what you mean by `both_factor` here, why is it better than `second_term`?

Reply:
Ah it is not for better, It is just because I did not use `first_term` anymore, so I changed `second_term` into `both_factor` meaning the factor appears in both road and roof, and to avoid misleading that there is another `first_term`.

Reply:
Would you like me to change back to `second_term`?

Reply:
Maybe one of these options or, something similar?

`first_term` and `second_term`
OR
`surface_specific_factor` and `both_factor`